regression/scripts/release_cray.sh

```diff
@@ -98,15 +98,15 @@ export TARGET=knl
 ##---------------------------------------------------------------------------##
 sdir=`dirname $0`
 cdir=`pwd`
```

Comment:
I think everything looks good. One comment that shouldn't hold up the merge: it'd be nice if some of these variables had more descriptive names. What are "sdir" and "cdir"?